ss58-registry.json

```diff
@@ -550,6 +550,24 @@
       "standardAccount": "*25519",
       "website": "https://sora.org"
     },
+    {
+      "prefix": 71,
+      "network": "3dp",
+      "displayName": "3DP network",
+      "symbols": ["3DP"],
+      "decimals": [12],
+      "standardAccount": "*25519",
+      "website": "https://3dpass.org"
+    },
+    {
+      "prefix": 72,
+      "network": "3dpt",
```

Comment:
"network" needs to be a valid rust identifier, e.g. not starting with 3. Please fix this and ensure that the crate compiles locally.

Reply:
Unfortunately, we can not use different network identifier, because we'll have to change the "symbols" as well. As we were trying to compile the project with the identifier "t3dp", it causes the error with the symbol. It doesn't correspond to our WP and the network running. We just can not afford it to do hard fork for this reason. Hope you understand.

Reply:
> It doesn't correspond to our WP

What does WP stands for?

Reply:
3DPass official White Paper: https://3dpass.org/3DPass_white_paper.pdf

Reply:
Eventually, we found some possibilities to rename our symbols to P3D and P3Dt. Hope, it's OK now.